Update README with "Inactive, but open for PRs" message (#316)
strangelove-ventures/lens: README.md

# `lens`

## Inactive :

> The "lens" repository, developed by Strangelove, is now inactive. It serves as a command-line tool and library to interact with Cosmos chains, supporting the core Cosmos-SDK modules.This repo will be kept open for PRs but will no longer be actively developerd by Strangelove.

Reviewer: ```suggestion
This repository is now inactive. It will be kept open (non-archived) but will no longer be actively developed by Strangelove.
```

# `lens`

## Archival Notice

> The "lens" repository, developed by Strangelove Ventures, is now archived. It served as a command-line tool and library to interact with Cosmos chains, supporting the core Cosmos-SDK modules.

Reviewer: ```suggestion
The "lens" repository, developed by Strangelove, is now archived. It served as a command-line tool and library to interact with Cosmos chains, supporting the core Cosmos-SDK modules.
```

Reviewer: Second sentence here is redundant to lines 14-16

> The repository was divided into two main packages: `cmd`, which housed the `lens` command-line tool, and `client`, comprising foundational elements to develop Cosmos clients in Go.

Reviewer: This sentence is redundant to existing content in this file.

> It found utility in various scenarios like trading bots, server-side integrations, indexers, search engines, and transaction automation for farming applications.

Reviewer: This sentence is also redundant

> This repository is no longer active. For ongoing development or usage of Cosmos client libraries, please refer to the latest resources.

Reviewer: What latest resources?
